### DISCUSSION TITLE
同一文件当前只被采集处理一次

### DISCUSSION THREAD

**Discussion by XTUxiongda** [alibaba/ilogtail#1340] [Help] [answered]
<img width="724" alt="fbdf1baa0c22c10ec04ebd8b19e79f6" src="https://github.com/alibaba/ilogtail/assets/38322373/66a45693-5b3f-407c-ba0c-ca8cb4c47fdc">

我现在有两个配置文件，一个是采集ERROR日志，一个是采集INFO日志，我发现ERROR日志这个处理器扫过这个日志文件后，INFO的就不处理了。

问题：如果我想配多个配置文件，一个过滤ERROR日志去kafka发topic1,一个过滤INFO日志去kafka发topic2，我发现被ERROR处理完，INFO那边就不处理了

**Comment by 7y-9** [comment, marked as answer]
AllowingIncludedByMultiConfigs | bool | 否 | false | 是否允许当前配置采集其它配置已匹配的文件。
-- | -- | -- | -- | --

https://ilogtail.gitbook.io/ilogtail-docs/plugins/input/input-file
看下这个配置
